helm/charts/oathkeeper-maester/templates/deployment.yaml

```diff
@@ -51,14 +51,20 @@ spec:
             {{- if .Values.rulesFileName }}
             - --rulesFileName={{ .Values.rulesFileName }}
             {{- end }}
+          env:
+            - name: NAMESPACE
+              {{- if .Values.singleNamespaceMode }}
+              value: {{ .Release.Namespace }}
+              {{- else }}
+              value: ""
+              {{- end }}
```

Comment:
We don't need to duplicate os.getenv behaviour here, let's set NS env if it's enabled by the flag